base/util_test.go

```diff
@@ -1735,3 +1735,33 @@ func TestCASToLittleEndianHex(t *testing.T) {
 	littleEndianHex := Uint64CASToLittleEndianHex(casValue)
 	require.Equal(t, expHexValue, string(littleEndianHex))
 }
+
+func TestUint64CASToLittleEndianHexAndStripZeros(t *testing.T) {
+	hexLE := "0x0000000000000000"
+	u64 := HexCasToUint64(hexLE)
+	hexLEStripped := Uint64ToLittleEndianHexAndStripZeros(u64)
```

Comment:
Running this test locally, the value of hexLEStripped is "000".  Is this what we intend?
It looks like any hex value with only two non-zero digits retains an extra zero.  (e.g. `0xa500000000000000` results in `a50` when I would have expected `a5`.  Is there a requirement for a minimum length of three?

Reply:
Good spot, fixed this so now that delta is just 0